src/js/css.js

```diff
@@ -9,6 +9,12 @@ export function injectCSS(css) {
     const style = document.createElement('style')
     style.type = 'text/css'
     style.textContent = css
-    document.head.insertBefore(style, document.head.firstChild)
+    const head = document.head
+    const { firstChild } = head
+    if (firstChild) {
+      head.insertBefore(style, firstChild)
+    } else {
+      head.appendChild(style)
```

Comment:
Why not just append in all cases?

Reply:
We want to append to the top of the `<head>` so styles can be overridden by the user's stylesheets below it